roles/keystone/tasks/saml.yml

```diff
@@ -21,7 +21,7 @@
 - name: Create idp metadata file
   template: src=etc/shibboleth/idp_metadata.xml
             dest=/etc/shibboleth/idp_metadata.xml
-  when: keystone.federation.sp.saml.providers[0].metadata_provider_type == 'file'
+  when: keystone.federation.sp.saml.providers[0].metadata_provider_type|default('url') == 'file'
```

Comment:
so what happens when metadata_provider_type = 'url'? How is /etc/shibboleth/idp_metadata.xml written out?